app/main/views/services.py

```diff
@@ -37,7 +37,13 @@ def list_services():
 
     supplier_id = request.args.get('supplier_id')
 
-    services = Service.query.filter(Service.status == 'published')
+    services = Service.query
```

Comment:
We need a story for the framework stuff.

I figures APIs return everything on the GET by Id and the GET list queries, with limits on the list endpoint if needed. We're the client and should configure accordingly.

In the future we should give our API tokens roles and deal with it there.